packages/components/psammead-most-read/README.md

```diff
@@ -4,6 +4,8 @@
 
 The `MostRead` component is designed to display the most read articles given a designated period of time which is dependent on service. The component comprises of a `MostReadList` containing an ordered list of `MostReadItemWrapper`. A `MostReadItemWrapper` is a list item comprised of and a `MostReadLink`, a link to the article, and a `MostReadRank` which is a numerical counter representing the article's ranking.
 
+The `MostReadLink` and `MostReadRank` components also support different a size prop. This prop currently only accepts a `default` and `small` size. Internally these values change the typography used for the components.
```

Comment:
In "Internally these values change the typography used for the components.", can we focus less on the typography and more on the CSS? Since we're also changing other things, like padding.

I think this might help people understand that `size` relates to more than just the typography and more to the component's overall 'theme'.